### PR TITLE
Adding codecov action to workflows

### PR DESCRIPTION
**Pull request by mamankhan99**
codecov action added to the ci that uploads coverage report to [codecov.io](https://codecov.io)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,3 +27,5 @@ jobs:
       run: npm run lint
     - name: Test
       run: npm run test
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```
Comment (arbrandes) [inline .github/workflows/ci.yml:31]: Nitpick: can we get a newline at the end of the file?

**Review by arbrandes — CHANGES_REQUESTED** [review]
A very, very minor nitpick: otherwise, good to go.

**Comment by jmbowman** [comment]
@arbrandes Looks like the requested change was made, do you want to update your review?  It's currently blocking merging.

**Review by arbrandes — APPROVED** [review]
Apologies for the delay.  Looks good to me!

**Pull request merged**